src/service.js

```diff
@@ -395,6 +406,9 @@ function getServiceInternal(holder, id) {
     if (s.resolve) {
       s.resolve(s.obj);
     }
+    if (s.reject) {
+      delete s.reject;
+    }
```

Comment:
I don't think this is necessary. Calling reject() is a no-op for an already resolved promise.